mappings/net/minecraft/client/gui/DrawableHelper.mapping

```diff
@@ -36,15 +40,15 @@ CLASS net/minecraft/class_332 net/minecraft/client/gui/DrawableHelper
 		ARG 5 v
 		ARG 6 uWidth
 		ARG 7 vHeight
-		ARG 8 texWidth
-		ARG 9 texHeight
+		ARG 8 textureWidth
+		ARG 9 textureHeight
 	METHOD method_25294 fill (IIIII)V
 		ARG 0 x1
 		ARG 1 y1
 		ARG 2 x2
 		ARG 3 y2
 		ARG 4 color
-	METHOD method_25295 innerBlit (IIIIIFFFF)V
+	METHOD method_25295 drawTexturedQuad (IIIIIFFFF)V
 		ARG 0 xStart
 		ARG 1 xEnd
 		ARG 2 yStart
```

Comment:
Are the xEnd and yStart in right order here?

Reply:
Yes.

Reply:
yes